src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\BinLookup;
+
+class Get3dsAvailability extends \Adyen\Service\AbstractResource
+{
+    /**
+     * @var string
+     */
+    protected $endpoint;
+
+    /**
+     * Include applicationInfo key in the request parameters
+     *
+     * @var bool
+     */
+    protected $allowApplicationInfo = true;
+
+    /**
+     * Authorise constructor.
+     *
+     * @param \Adyen\Service $service
+     */
+    public function __construct($service)
+    {
+        $this->endpoint = $service->getClient()->getConfig()->get('endpoint') . '/pal/servlet/BinLookup/' . $service->getClient()->getApiBinLookupVersion() . 'get3dsAvailability';
```

Comment:
```suggestion
        $this->endpoint = $service->getClient()->getConfig()->get('endpoint') . '/pal/servlet/BinLookup/' . $service->getClient()->getApiBinLookupVersion() . '/get3dsAvailability';
```

Sorry I missed a slash sign.
Everything else looks good 👍